vpr/src/route/route_common.h

```diff
@@ -38,6 +38,7 @@ struct t_heap {
 	float backward_path_cost = 0.;
 	float R_upstream = 0.;
 	int index = OPEN;
+        float T_upstream = 0.;
```

Comment:
We should make sure there is a comment describing what T_upstream represents, and why it is necessary.